source/_patterns/01-molecules/teaser/teaser~ar.yaml

```diff
@@ -0,0 +1,8 @@
+attributes:
+  dir: rtl
+  lang: ar
+heading:
+  text:
+    attributes:
+      href: '#'
```

Comment:
Still tempted to move this to the top level. (Saves duplicating for other clickable things, eg image.)

Reply:
Experimented in 2c78982125afe7b0ae68e9ccc743da88b3ff4db5.

Reply:
@thewilkybarkid Makes sense to me, as long as it doesn't preclude having more than one `href` value supplied to a pattern if necessary (and it doesn't look like it does).